src/components/SingleArticle.jsx

```diff
@@ -17,8 +17,8 @@ const SingleArticle = () => {
   return (
     <>
       <h1 data-cy="article-title">{article.title}</h1>
-      <h1 data-cy="article-body">{article.body}</h1>
```

Comment:
In the "show" action of the article we get the full article body, we don't display a teaser.
From api:
{"article"=>
  {"id"=>206,
   "title"=>"Hello World",
   "body"=>"The world is a strange place",
   "category"=>"Mystring",
   "published"=>"2022-01-23T16:17:36.930Z"}}